src/components/AuthenticatedContent/AuthenticatedContent.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+// Define the type for the props
+interface AuthenticatedContentProps {
+  username: string;
+  email: string;
+  password: string;
+}
+
+// Use the defined type for the props
+const AuthenticatedContent: React.FC<AuthenticatedContentProps> = ({
+  username,
+  email,
+  password,
```

Comment:
We can remove the `password` entry, since we won't ever want to show that to the user